front/src/hooks.ts

```diff
@@ -0,0 +1,4 @@
+import { useDispatch } from 'react-redux';
+import { AppDispatch } from './redux/store';
+
+export const useAppDispatch: () => AppDispatch = useDispatch;
```

Comment:
what is the reason/need behind this wrapper?

Reply:
@skyboyer 
It creates typed dispatch for you and allows you dispatch a thunk ([link](https://redux-toolkit.js.org/usage/usage-with-typescript#getting-the-dispatch-type ))

Reply:
ah, I see. all is good then